CMakeLists.txt

```diff
@@ -61,13 +61,13 @@ file(WRITE ${CMAKE_BINARY_DIR}/coverage_targets.txt "")
 # Determine platform to build
 if(AZ_PLATFORM_IMPL STREQUAL "WIN32")
   # build windows platform
-  set(PAL az_win32)
+  set(PAL az_platform_win32)
 elseif(AZ_PLATFORM_IMPL STREQUAL "POSIX")
   # build linux platform
-  set(PAL az_posix)
+  set(PAL az_platform_posix)
 else()
   #noplatform
-  set(PAL az_noplatform)
+  set(PAL az_platform_stub)
```

Comment:
There should be some form of warning or error to inform the users that there are actions required here.

Reply:
I think it should build and be usable but I can output a message on CMake that lets the user know it is only using stubs?

Reply:
Last I've tried it two weeks ago (drag&drop the entire folder struct) it didn't work because of preconditions requiring the time PAL implemented.

I do agree with you: it should just work (and no warning in that case).

Reply:
Yes we will talk about the specifics of what changes we might want. I'm looking to send an email today and have a short meeting with our team tomorrow to formalize/discuss what we might want to change.